scripts/background.js

```diff
@@ -40,34 +40,47 @@ setInterval(() => {
                             { src: '/images/eshop-icon.png', sizes: '256x256', type: 'image/png' }
                         ]
                     });
-                };
+                }
             } else if (!regex.settings.test(url) && !regex.shopping.test(url) && !audio.paused && !audio.src.includes(audioSource.health)) {
-                audio.pause();
-            };
+                fadeOut();
+            }
         });
 }, 1000);
 
+function fadeOut() {
+    let volume = audio.volume;
+    const fadeOutInterval = setInterval(() => {
+        if (volume > 0) {
+            volume -= 0.001;
```

Comment:
I think subtracting by 0.001 is a bit slow and 0.01 sounds better.

```suggestion
            volume -= 0.01;
```